### PR TITLE
Issue 1

### LINKED ISSUE #1: See All Reminders
When the user visits the root of the application (e.g. the "/" URL), they should see a list of all of the reminders on the page.

- Each reminder should have a class of .spec-reminder-item.
- There is a failing test set up in tests/acceptance/reminder-list-test.js.

Hot Tip: If you put “Closes #1” (assuming the issue is marked #1) in the pull request, your commit, or anywhere in the body of your PR, it will automatically close this issue when it successfully merged into master.

### PR DESCRIPTION
**Pull request by rcwestlake**
@martensonbj @tman22 @stevekinney @nfosterky

We made a Reminder route, template, controller, and model. In the model, we make the API call and that gets passed to the template where you use "each" to render each title to the page with the appropriate class. 

http://recordit.co/BNrQVVDAwm

This code closes #1.

### CODE REVIEW

File: app/models/reminder.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  title: DS.attr('string'),
+  date: DS.attr('string'),
```
Comment (martensonbj) [inline app/models/reminder.js]: Dig into the docs to find a better data type for the date property...check out [this example](https://guides.emberjs.com/v2.10.0/models/defining-models/#toc_transforms)

File: app/routes/index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+});
```
Comment (martensonbj) [inline app/routes/index.js]: Let's get rid of extra files that aren't being used.

File: tests/unit/routes/index-test.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+moduleFor('route:index', 'Unit | Route | index', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+});
+
+test('it exists', function(assert) {
+  let route = this.subject();
+  assert.ok(route);
+});
```
Comment (martensonbj) [inline tests/unit/routes/index-test.js]: Since we don't need the `routes/index.js` file we don't need this file either.

**Comment by rcwestlake** [comment]
Thanks @martensonbj. The changes have been made. I pushed up to the branch and see that the commit message is in this thread. ⛵️ 

@hilarylewis92, see above

**Comment by martensonbj** [comment]
🧀

**Comment by stevekinney** [comment]
You got this a bit more right in #2, but I would just name the PR based on the fature you're implementing. You can always says "Closes #1" in the title and it will automatically close the issue.

**Pull request merged**